build.gradle

```diff
@@ -322,5 +331,48 @@ task buildMap(type: JavaExec, dependsOn: 'build') {
     classpath = project.sourceSets.main.runtimeClasspath
 }
 
+task codeCoverageReport(type: JacocoReport) {
```

Comment:
This adds the codeCoverageReport command to gradlew. Which will generate the html report in build/report/jacoco/codecoverage/ (that path isn't exact). Again I don't really know anything about gradle or jacoco. I just tried a bunch of different options each times trying to get less errors until it worked.